### DISCUSSION TITLE
Stats collection and presentation

### DISCUSSION THREAD

**Discussion by lmilbaum** [renovatebot/renovate#23953] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

The following stats might be of interest:
How many repos are onboarded vs total repos?
How many PRs opened, merged, closed?
Merged list of dependencies

We would like to use a dashboard to present those stats.

Was something similar implemented successfully?

### What you tried so far.

Nothing much so far

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by secustor** [comment, marked as answer]
https://github.com/renovatebot/renovate/discussions/23105
